R/derive_param_wbc_abs.R

```diff
@@ -0,0 +1,172 @@
+#' Add a parameter for lab differentials converted to absolute values
+#'
+#' @description
+#'
+#' Add a parameter by converting lab differentials from percentage to absolute values
```

Comment:
Should be "from fraction or percentage to absolute values"

Reply:
Updated.